### ISSUE TITLE
Add BackUp to Otel suite of feedstocks

### ISSUE THREAD

**Issue by rxm7706** [conda-forge/opentelemetry-sdk-feedstock#34]
### Comment:

https://github.com/conda-forge/opentelemetry-instrumentation-aiopg-feedstock/pull/21

**Comment by rxm7706** [comment]
Full list of all Opentel Packages 


![283998176-49f91ac5-5637-4f2e-ac62-b8586b4f6838](https://github.com/conda-forge/opentelemetry-sdk-feedstock/assets/95496360/8c31a4ab-b362-403f-b51b-bd8e829ff6e5)

**Comment by hmaarrfk** [comment]
The thing I'm confused about is that you are making a request to be added as a maintainer, but you are listing specific problems that exist.

Can you summarize it in 1 sentence?

Finally, I would like to go a different way, you have listed the different "stages" you need to go through.

Please make PRs to the remaining level 0 feedstocks implementing the fixes:

1. Accomplishes the concrete fix you need.
2. Adds yourself to all the level 0 feedstock.

For the later stages, we can add the whole  to the maintainers of one of the level0 feedstocks to avoid this kind of future problem.

https://github.com/conda-forge/qt-feedstock/blob/main/recipe/meta.yaml#L91

You would:

1. Make the concrete fix you need.
2. Add a maintainer
4. Ping us for a single PR.

Otherwise, it is a little difficult for us to judge exactly how you will address the problems at hand.

Ultimately, I don't like to entertain the requests: "please just add me". I think it goes a long way to "fix something" then "add yourself as a commitment to continued maintenance".

**Comment by rxm7706** [comment]
> Accomplishes the concrete fix you need.
> "fix something" then "add yourself as a commitment to continued maintenance".

Hi @hmaarrfk 

Thank you for taking the time to respond. I was operating on the assumption that my intent was apparent. My Apologies. 

- All 50+ feedstocks for open telemetry have only one maintainer mariusvniekerk; who is a key contributor to conda-forge, working on more important items with other core / admins. I've updated the title too that shows all the add maintainer PRs together.
- Because of the levels of dependency - this suite ends up needing custom analysis and sequential re-execution every release - I have made PR's and rerun, waited for weeks for an admin to merge for the last 3 release this year. 
- I currently own and have helped maintain over 200+ feedstocks on conda-forge - so I'm happy to help where I can. 

_My thought process was - the simplest contribution I could made was figure out the dependency and take rerun's, the easy fixes of dependency version changes and keep this suite uptodate for everyone. My list of ten PRs to add me as co-maintainer - addresses the packages the rest depend  on - and I believe If I can keep those 10 uptodate, the rest are easier to tackle, and I have PRs open to add me as an alternate maintainer linked above_

I would like to perhaps consolidate all Level 0's in this repo, or make this entire suite subpackages like airflow - there is another issue proposing that. That might take a bit more time, and I would probably need more direction and help before I attempt that. 

For one concrete next steps
opentelemetry-exporter-otlp is at version 1.18.0 while the rest of the suite - while I have been able to bring bits and pieces of the rest of the suite to version 1.21.0
If you can merge this
https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/16
I have https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/14 ready to merge, and it clear one big hurdle for Conda Airflow feedstock users - where pip install brings in version 1.21.0 and conda install brings in version 1.18.0

Please let me know if you need any further clarification.
Thanks & Regards

**Comment by hmaarrfk** [comment]
Please take some time to read my message and make my life easier to help you.

Thank you for mentioning that you are active in conda-forge, but it is a rather big community, and we may not have overlapped. I checked your github profile and in fact you have forked a bunch 

I was the member of core that responded to your request. So please take a moment to respond to mine.

I personally do not like to merge PRs that just add maintainers. How does that help your workflow? You will open a PR anyway with the rest of the changes later.

In my mind you can:

1. clone a bunch of repos
2. fix all of them
3. add yourself to all of them
4. use `mamba build` to build them locally
5. then really gain confidence they are all fixed
6. then me on the PRs that are green ,with your name on them. 

Please get https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/14 to a working state, and add yourself as a maintainer there. Then when you are ready ping me.

I prefer to merge https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/14, not https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/16


Other core members may have a different flow, especially if they have worked with you before. but this is mine. please respect it.

**Comment by rxm7706** [comment]
> Other core members may have a different flow, especially if they have worked with you before. but this is mine. please respect it.

@hmaarrfk absolutely - I have only been contributing for <2 years. Glad to take direction from you. Appreciate you taking the time. 

I've already realizing my inefficiencies. I have made so many repetitive PRs every 3 months to Feedstocks - Because I was skipping step 3 in your instructions. So I've been pinging and waiting weeks each time - vs adding myself as a maintainer while fixing the feedstock. 

I was not sure of the right etiquette and thought requesting to be added to a feedstock as a maintainer should always be a separate request. 

 - https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/14 has been fixed for your review. 
 - I've added myself as a maintainer but was not able to close https://github.com/conda-forge/opentelemetry-exporter-otlp-feedstock/pull/16 (which was created to add me as a maintainer - and is now duplicate. )

**Comment by hmaarrfk** [comment]
I think for me the right etiquette is to ask in the PR if you can be added. typically mentioning other feedstocks you work on is a good way to build credibility. 

if somebody doesn't respond within 1-2 weeks, then you can go ahead and add yourself, and ping core.

that way, within 2 weeks you would have fixed the repo, and helped maintain it in a slightly longer run.

the main idea, is to 

1. fix first
2. ask to be part of the longer term maintenance once you have fixed something meaningful.

meaningful varries, but a version bump definitely qualifies.

**Comment by hmaarrfk** [comment]
Ok i decided to help out a little bit and I merge the "needed for airflow" ones.

Please do consider adding `conda-forge/opentelemetry-api` or `conda-forge/opentelemetry-sdk` as a maintainer to the future feedstock PRs you make:

https://github.com/conda-forge/qt-feedstock/blob/main/recipe/meta.yaml#L91
